assets/js/blocks/cart/inner-blocks/cart-accepted-payment-methods-block/block.json

```diff
@@ -13,5 +13,6 @@
 	},
 	"parent": [ "woocommerce/cart-totals-block" ],
 	"textdomain": "woo-gutenberg-products-block",
+	"$schema": "https://schemas.wp.org/trunk/block.json",
```

Comment:
I forget to ask why was this needed?

Reply:
Loading the [schema definition](schema definition file. Supported editors can provide help like tooltips, autocomplete, and schema validation.) file allow supported editors to provide help like tooltips, autocomplete, and schema validation. For example, you can hover over a field in `block.json` to see a tooltip description for it. So, it's always good to add this line for development purposes.